_guides/guides.md

```diff
@@ -1,5 +1,5 @@
 ---
 layout: guides-index
-title: Quarkus - Guides
+title: Guides - Latest
```

Comment:
I wonder if we should have this done dynamically in the template and maybe with a badge of some sort (we could do it for all the versions in one go)?

I can do the code if you can just do the layout part with a badge for the version.